README.md

```diff
@@ -331,18 +331,19 @@ Please note that it is possible to pass argument variables within the command
 line to instrument the build phase. As an example, the following is a possible
 workflow to configure and build a customized 10.5 instance:
 
-``` sh
-pgenv config write 10.5
+``` 
+$ pgenv config write 10.5
 pgenv config edit 10.5
 # adjust PGENV_CONFIGURE_OPTIONS
-pgenv build 10.5
+
+$ pgenv build 10.5
 ```
 
 In the case you need to specify a particular variable, such as the Perl
 interpreter, pass it on the command line at the time of build:
 
 ``` sh
-PERL=/usr/local/my-fancy-perl pgenv build 10.5
+$ PERL=/usr/local/my-fancy-perl pgenv build 10.5
 ```
```

Comment:
This one is pure shell, so omit the `$` one keep the `sh`.

Reply:
Fix in beb5f98